package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "datadog-cdk-constructs",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
You don't need to increment the version number here unless you are actually publishing the changes to npm as a new release. Actually, I think we would ideally change the version number in the `package.json` as part of the release script.